.github/workflows/build.yml

```
@@ -85,7 +85,7 @@ jobs:
       - name: Install Terraform-docs
         env:
           PACKAGE_URL: github.com/terraform-docs/terraform-docs
-          PACKAGE_VERSION: ${{ steps.setup-env.outputs.terraform-docs-version }}
+          PACKAGE_VERSION: ${{ steps.setup-env. outputs.terraform-docs-version }}
```

Pretty sure this space shouldn't be here:
```suggestion
          PACKAGE_VERSION: ${{ steps.setup-env.outputs.terraform-docs-version }}
```